src/Tools/CoreFx.Tools/FindBestConfiguration.cs

```diff
@@ -0,0 +1,119 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Utilities;
+using Microsoft.Build.Framework;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public class FindBestConfiguration : BuildTask
+    {
+        private static readonly char s_SupportedGroupsSeparator = '|';
+        private static readonly char s_VerticalGroupsSeparator = '-';
```

Comment:
I'm guessing it won't make a measurable difference, but since you already have these as static readonly fields, you could change them from `char` to pre-allocated `char[]` arrays, to avoid the params array allocations when used with `Split`, e.g.:

```c#
private static readonly char[] s_SupportedGroupsSeparator = { '|' };
private static readonly char[] s_VerticalGroupsSeparator = { '-' };
```